openmdao/core/system.py

```diff
@@ -535,7 +541,7 @@ def fd_jacobian(self, params, unknowns, resids, total_derivs=False,
 
                         # delta resid is delta unknown
                         resultvec.vec[:] -= cache1
-                        resultvec.vec[:] *= (1.0/step)
```

Comment:
I just did some profiling. Multiplying a large vector by 1.0/step instead of dividing the vector by step is much faster, so this should be reverted.